### PR TITLE
ASSMOS PART 1[WIP]

### PR DESCRIPTION
**Pull request by as334**
:cl: optional name here
add: Added 2 new gases, Ionized Plasma and Ozone
add: Added Electricity to gas mixtures. Adding electricity to gases will cause new reaction to happen.
/:cl:

Heavily a work in progress. 

To do:
- [ ] Ways to electrify gas
- [ ] Tile actions for electrified gas(lightning discharges)
- [ ] Rework single tank bombs
- [ ] Possibly add Brown's Gas(Nitric Oxide)
- [ ] Fix inevitable bugs and balance concerns.
- [ ] Find more Taylor Swift for coding music

The rough systems for the gas reactions and interactions at the gas mixtures level are complete though, and I'd like feedback on those.

@duncathan

### CODE REVIEW

**Comment by TechnoAlchemisto** [comment]
gas bloat

**Comment by as334** [comment]
it's not bloat if you have to make them intentionally.

**Comment by bgobandit** [comment]
we already have nitrous oxide...

File: code/modules/atmospherics/gasmixtures/gas_mixture.dm [hunk 1]
```diff
@@ -111,10 +112,10 @@ var/list/gaslist_cache = null
 		return total_moles() * R_IDEAL_GAS_EQUATION * temperature / volume
 	return 0
 
-/datum/gas_mixture/proc/return_temperature() //kelvins
+/datum/gas_mixture/proc/return_temperature() //kelvin
 	return temperature
 
-/datum/gas_mixture/proc/return_volume() //liters
+/datum/gas_mixture/proc/return_volume() //litres
```
Comment (MMMiracles) [inline code/modules/atmospherics/gasmixtures/gas_mixture.dm]: GO HOME REST OF THE WORLD

USA USA USA USA

**Comment by as334** [comment]
Oh yeah my bad. Nitric Oxide. One oxygen not 2.

File: code/modules/atmospherics/gasmixtures/gas_mixture.dm (hunk 1 above)
Comment (as334) [inline code/modules/atmospherics/gasmixtures/gas_mixture.dm]: Our units, Our spelling. Git out.

Comment (Iamgoofball) [inline code/modules/atmospherics/gasmixtures/gas_mixture.dm]: U S A

Comment (Niknakflak) [inline code/modules/atmospherics/gasmixtures/gas_mixture.dm]: U S A

Comment (PKPenguin321) [inline code/modules/atmospherics/gasmixtures/gas_mixture.dm]: :us:

Comment (MMMiracles) [inline code/modules/atmospherics/gasmixtures/gas_mixture.dm]: :us: freedom brigade stands strong:us:

**Comment by Iamgoofball** [comment]
U S A

File: code/modules/atmospherics/gasmixtures/gas_mixture.dm (hunk 1 above)
Comment (Iamgoofball) [inline code/modules/atmospherics/gasmixtures/gas_mixture.dm]: :us:

File: code/__DEFINES/atmospherics.dm [hunk 2]
```diff
@@ -12,6 +12,7 @@
 #define MOLES			1
 #define ARCHIVE			2
 #define GAS_META		3
+#define SPECIAL			4
```
Comment (MrPerson) [inline code/__DEFINES/atmospherics.dm]: Calling it "SPECIAL" is almost as useless as just leaving it as 4. Come up with a more descriptive name or at least leave a helpful comment

**Comment by AdamElTablawy** [comment]
So, what does ozone and ionized plasma DO exactly

**Comment by as334** [comment]
Ozone at the moment is made by introducing an electric current to oxygen and when exposed to Freon gets broken up back into oxygen, releasing heat. In the future it'll also be toxic, and in followup PRs will be used more often as a catalyst in reactions. Ionized Plasma is created by super heating Plasma. It gradually soaks up heat from its environment, until you apply an electric current to it, at which point it releases all the energy violently.

**Comment by NocturnalQuill** [comment]
Yes to ionized gasses, no to new gasses. I love the idea of being able to modify existing gasses through ionization. However, two new gasses is enough for a year. We already need to rearrange atmos to accommodate tanks for the new gasses as it is.

**Comment by Razharas** [comment]
>electricity gas

0/10000000000000000000000000000000000000000000000000000000

**Comment by as334** [comment]
Razharas trust me it's better than the alternative of having to add a new snowflake variable and handling system to all of atmos. Also @NocturnalQuill these gases are designed not to be roundstart, they have to be synthesized. Believe it or not, not every gas has to have its own tank.

**Comment by as334** [comment]
I've been away from my computer till tuesday, will continue work on PR then.

**Comment by Jalleo** [comment]
I cant wait for these gases.

**Comment by as334** [comment]
Closed until my computer stops being hilariously ass.

**Comment by duncathan** [comment]
@as334 one day I will return for you, my sweet

**Pull request closed without merging**